### DISCUSSION TITLE
Move docs to seperate repository

### DISCUSSION THREAD

**Discussion by Scarjit** [united-manufacturing-hub/united-manufacturing-hub#709] [Ideas]
Docs should be moved to a separate repository.
This allows us to publish blog articles without disrupting the developers workflow / publishing new versions.

**Comment by JeremyTheocharis** [comment]
We would need a strategy then to ensure that all newly added code is documented in the other repository as well. Any ideas?

**Reply by Scarjit** [reply]
Our new pull request template has a checkbox " I have made corresponding changes to the documentation".
While this is no actual check, it is a reminder for devs (This is more then we currently have).

We could also add godoc to our CI pipeline to build documentation for the actual code, and even include an automatic pull request to our docs repo, to add any changed godoc html files in there.

**Reply by JeremyTheocharis** [reply]
We could let us inspire from these large doc repos:

1. https://github.com/MicrosoftDocs/azure-docs
2. https://github.com/github/docs

Idea 1: we could add versions to the documentation. with this one could quickly fix issues for the current version while working on the version of a future release.
Idea 2: we could require developers to create two PRs: one for this repo and one for docs. only if both of them are approved they can be merged.

**Comment by Scarjit** [comment]
I think having doc versions for all releases (0.1.0, ....) and one for latest (whatever is currently on the main branch of the docs repo would be nice)
Also i would be fine with 2 PRs, that would make it in my opinion easier to work with.
